eva_cttv_pipeline/evidence_string_generation/clinvar_to_evidence_strings.py

```diff
@@ -305,6 +310,15 @@ def get_consequence_types(clinvar_record_measure, consequence_type_dict):
         # Example of such an identifier: 14:23423715:G:A
         coord_id = ':'.join([clinvar_record_measure.chr, str(clinvar_record_measure.vcf_pos),
                              clinvar_record_measure.vcf_ref, clinvar_record_measure.vcf_alt])
+        # Non-ATGC allele sequences are not accepted by Open Targets schema 2.0.5
+        # Example: 12_32625716_G_H (from RCV000032000)
+        if ILLEGAL_ALLELE_SEQUENCE.search(clinvar_record_measure.vcf_ref + clinvar_record_measure.vcf_alt):
+            logger.warning(f'Skipping variant with non-ACGT allele sequences: {coord_id}')
+            return []
+        # FIXME: Open Targets schema 2.0.5 does not support mitochondrial variants, so they are not being processed
+        if clinvar_record_measure.chr == 'MT':
+            logger.warning(f'Skipping mitochondrial variant: {coord_id}')
+            return []
```

Comment:
Not sure it's important but these two categories of variant will be counted as `n_no_variant_to_ensg_mapping` where they have been filtered out for a different reason.

Reply:
That's right, thank you for pointing this out. I actually expect the exclusion of mitochondrial variants to be very much temporary, it's probably just an oversight in the schema which caused them to be forgotten.

Otherwise, the whole reporting & quality metric system is a mess right now. While almost everything else was refactored (sometimes multiple times) in the past two years, the reporting system is something which I haven't touched at all yet. There's a ticket https://github.com/EBIvariation/eva-opentargets/issues/114 to do this, I hope I can manage to do this before the end of March. I'll add your comment to that ticket.